babel.config.json

```
@@ -1,9 +1,13 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "presets": [
+        ["@babel/preset-env", {"useBuiltIns": "usage", "corejs": "3.18"}],
```

without setting the `useBuiltIns` option, `@babel/preset-env` will only transform new syntax (eg. [optional chaining syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)), but not polyfills for features like [`String.prototype.includes()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/String/includes)

---

interestingly, setting the `useBuiltIns` option does not add any polyfills to our build at the moment. the reason for this is that our `.browserslistrc` contains only modern browsers that support all of the features used in our code (and that the [`op_mini` target is ignored by `@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env#ineffective-browserslist-queries)).

---

to test that the feature works, you can add `chrome 80` to the .browserslistrc. with this target, the `es.array.reduce ` polyfill is included in the build. it is also possible to see all included polyfills by enabling the `debug` option of `@babel/preset-env`:
```suggestion
        ["@babel/preset-env", {"useBuiltIns": "usage", "corejs": "3.18", "debug": true}],
```